usr/local/share/bastille/templates/default/thick/Bastillefile

```diff
@@ -0,0 +1 @@
+INCLUDE default/thin
```

Comment:
This could reuse the `bastille_template_thin` variable. With that the thin template could also be configured.

Reply:
We don't have access to that variable inside the template, but I refactored it to have a `default/base` template to address this. Now a user can override/extend the base, which affects thin, thick, and vnet jails by default, or override/extend any of the specific templates for those jail types (including changing whether or not they extend `default/base`).

Reply:
I just tried to implement my own template and got your point. ;) I've made an update to pass the base template as an arg, so now you can set `bastille_template_base` to a custom template and it will be included by the thick, thin, and vnet templates without having to modify them. In my case, I created a base template that extends `default/base`, so it's as if I wedged changes in between.